L103-python-add-typing-to-sync-api.md

```diff
@@ -70,9 +63,11 @@ def __call__(self,
 
 ## Implementation
 
-1. Add typing to internal APIs.
-2. Add typing to public API and test though pre-release.
-3. Add typing to public APIs.
+* Add typing to internal APIs.
+  * [PR1: Add typing for some internal python files.](https://github.com/grpc/grpc/pull/31514)
+* Add typing to public API and test though pre-release.
+  * [PR link holder]
+* Release typed public APIs.
```

Comment:
Consider writing a blog post about this on [`grpc.io`](https://grpc.io/blog/) once it's finished. 

CC @tmwilson